pithos/pithos.py

```diff
@@ -415,6 +433,7 @@ def set_audio_quality(self, *ignore):
         self.worker_run('set_audio_quality', (self.settings.get_string('audio-quality'),))
 
     def pandora_connect(self, *ignore, message="Logging in...", callback=None):
+        self.connected = False
```

Comment:
> Wouldn't think so, though it would make this all a lot cleaner.

So do you want to go ahead and make Pandora a GObject? It would probably simplify a lot of stuff in the long run.